src/main/java/de/tum/cit/aet/artemis/exercise/repository/ExerciseMetricsRepository.java

```diff
@@ -48,8 +48,9 @@ CASE WHEN TYPE(e) = ProgrammingExercise THEN TREAT(e AS ProgrammingExercise).all
             )
             FROM Exercise e
             WHERE e.course.id = :courseId
+                AND COALESCE(e.startDate, e.releaseDate) <= CURRENT_TIMESTAMP
```

Comment:
Should we consider exercises that don't have a startDate and don't have a releaseDate "started"?